pkg/k8s/apis/cilium.io/v2/types.go

```diff
@@ -767,7 +767,7 @@ type ENISpec struct {
 	// associated instance is terminated
 	//
 	// +optional
-	DeleteOnTermination bool `json:"delete-on-termination,omitempty"`
+	DeleteOnTermination *bool `json:"delete-on-termination,omitempty"`
```

Comment:
This might require `make generate-k8s-api`

Reply:
I'll do that
It's updating the CRD? I assume it's only based on the json definition right?

Reply:
It was to update the deep copy. I don't believe the json definition needs to be updated.